gradle/libs.versions.toml

```diff
@@ -16,6 +16,8 @@ kotlin = "1.9.22"
 decompose = "3.0.0-alpha05"
 mvikotlin = "4.0.0-alpha02"
 serialization = "1.6.2"
+essenty-lifecycle-coroutines = "2.0.0-alpha02"
+coroutines-swing = "1.8.0"
```

Comment:
Let's just name it coroutines, as this version could be applicable to all coroutines dependencies in the future.